pipelines/k8s-test/Jenkinsfile

```diff
@@ -0,0 +1,250 @@
+        sh "aws configure set aws_secret_access_key $secret_key"
+        }
+    }
+    stage('Select Avaliable Cluster') {
+        def unavailable = sh (script:"az aks list | grep 'resourceGroup\": \"aks-' | cut -d '\"' -f 4", returnStdout: true).split()
+        clusters = clusters - unavailable
+        target_cluster = clusters[Math.abs( new Random().nextInt() % (0 - clusters.size()) )]
+    }
+
+    stage('Create Cluster') {
+        build job: '../Kubernetes/KubernetesDeploy/azure/master', parameters: [
+            [$class: 'StringParameterValue', name: 'action', value: "create"],
+            [$class: 'StringParameterValue', name: 'environment', value: target_cluster],
+            [$class: 'StringParameterValue', name: 'kubernetes_version', value: "1.14.8"],
+            [$class: 'StringParameterValue', name: 'enable_cloud_manager', value: "true"],
+            [$class: 'StringParameterValue', name: 'enable_tick', value: "true"],
+            [$class: 'StringParameterValue', name: 'enable_elastic_stack', value: "true"],
+            [$class: 'StringParameterValue', name: 'core_vmsize', value: "Standard_D4s_v3"],
+            [$class: 'StringParameterValue', name: 'core_max_vmcount', value: "3"],
+            [$class: 'StringParameterValue', name: 'db_vmsize', value: "Standard_D16s_v3"],
+            [$class: 'StringParameterValue', name: 'meta_max_vmcount', value: "3"],
+            [$class: 'StringParameterValue', name: 'meta_vmsize', value: "Standard_D8s_v3"],
+            [$class: 'StringParameterValue', name: 'db_max_vmcount', value: "12"],
+            [$class: 'StringParameterValue', name: 'tag_creator', value: scmVars.GIT_BRANCH]
+        ]
+    }
+    stage('Create Splice-DB') {
+        dir ('dbaas-infrastructure/kubernetes/') {
+            sh "ls"
+            sh "az aks get-credentials --resource-group ${target_cluster} --name ${target_cluster}"
+            sh "helm init --client-only"
+            sh "kubectl get pods -A"
+            sh "./create-splice-operator.sh ${target_cluster} az default"
+            sh "kubectl get splicedbclusters"
+            sh "kubectl get po -n test"
+        }
+    }
+    stage('Run Integration Tests') {
+        wrap([$class: 'VaultBuildWrapper', vaultSecrets: vaultSecrets]) {
+            dir ('dbaas-infrastructure/kubernetes/') {
+                checkSpliceDB()
+                def jdbc_ip = sh (script:"kubectl get svc -A | grep 'LoadBalancer.*1527' | awk '{\$1=\$1};1' | cut -d \" \" -f 5 | xargs echo -n", returnStdout: true)
+                if ( jdbc_ip == "") {
+                    throw exception("JDBC URL not found/initialized")
+                }
+                sh "helm install --name integration-test charts/integration-tests --namespace test --set jdbc_url='jdbc:splice://$jdbc_ip:1527/splicedb;ssl=basic;password=admin;' --set splice_release='DB-9133_3.0' --set artifact_user='$jenkins_user' --set artifact_password='$jenkins_pass'"
```

Comment:
DB-9133-3.0? that should maybe be a variable, no?

Reply:
Updated it to set the release to the current branch that its pulling from. At the time, DB-9133 was the only branch that had working remote-it's.

Reply:
👍